server/src/middlewares/form.ts

```diff
@@ -0,0 +1,52 @@
+import type { RequestHandler } from "express";
+import Joi from "joi";
+
+const companySchema = Joi.object({
```

Comment:
Il faudrait rajouter dans le schema une propriété confirmPassword afin de comparer les deux passwords : 
https://stackoverflow.com/questions/29827082/hapi-route-joi-validation-of-password-confirmation